receiver/postgresqlreceiver/codegen.go

```diff
@@ -0,0 +1,17 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:generate mdatagen --experimental-gen metadata.yaml
```

Comment:
Future PR: I know we use this as the "standard" to do this, should we consider maybe to add this to "doc.go". We should have a "doc.go" anyway, so we can add this there?

Reply:
👍 Makes sense to me